app/recordtransfer/templates/includes/in_progress_submission_table.html

```diff
@@ -3,17 +3,29 @@
     <table class="full-width-table standard-table" cellspacing="0">
         <tr>
             <th>{% trans "Last Updated" %}</th>
+            <th>{% trans "Expires At" %}</th>
             <th>{% trans "Transfer Title" %}</th>
             <th>{% trans "Resume Transfer" %}</th>
             <th>{% trans "Delete" %}</th>
         </tr>
         {% for submission in in_progress_page_obj %}
             <tr>
                 <td>{{ submission.last_updated|date:"D M j, Y" }} @ {{ submission.last_updated|time:"H:i" }}</td>
-                <td>{{ submission.title }}</td>
                 <td>
-                    <a href="{% url 'recordtransfer:transfer' %}?transfer_uuid={{ submission.uuid }}">Resume</a>
+                    {% if submission.upload_session.expires_at %}
+                        <span class="{% if submission.expiring_soon %}red-text{% endif %} {% if submission.upload_session.expired %}strikethrough{% endif %}">
+                            {{ submission.upload_session.expires_at|date:"D M j, Y" }} @ {{ submission.upload_session.expires_at|time:"H:i" }}
+                        </span>
+                    {% else %}
+                        -
+                    {% endif %}
                 </td>
+                <td>{{ submission.title }}</td>
+                <td>
+                    {% if not submission.upload_session.expired %}
+                        <a href="{% url 'recordtransfer:transfer' %}?transfer_uuid={{ submission.uuid }}">Resume</a>
+                    {% endif %}
+                    </td>
```

Comment:
A dash when the upload session has expired would be suitable here. Also, Resume needs to be translatable.

```suggestion
                <td>
                    {% if not submission.upload_session_expired %}
                        <a href="{% url 'recordtransfer:transfer' %}?transfer_uuid={{ submission.uuid }}">
                            {% trans "Resume" %}
                        </a>
                    {% else %}
                        -
                    {% endif %}
                </td>
```

Reply:
Addressed by: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/405/commits/b58c4b748e549fd4e15577086879151716a4d992